src/api/ledger/pathfind.js

```diff
@@ -46,8 +46,12 @@ function requestPathFind(remote, pathfind: PathFind, callback) {
       _.omit(utils.common.toRippledAmount(amount), 'value'));
   }
 
-  remote.requestRipplePathFind(params,
-    composeAsync(_.partial(addParams, params), callback));
+  remote.createPathFind(params, function(data) {
+    if (data.full_reply) {
```

Comment:
All use cases should be passing the `onUpdate` function because without it I don't think you can guarantee that you will get the first `update` event. I just made it optional to avoid breaking things. I am tempted to just ignore that issue though and use a node-style callback.